### ISSUE TITLE
Remove the cpp11 tag on Github

### ISSUE THREAD

**Issue by vstumpf** [arun11299/cpp-jwt#81]
There's a `cpp11` tag in the About window of the repo, yet it doesn't actually support c++11.

Can you please remove it.
![image](https://user-images.githubusercontent.com/6844975/108944792-560a4500-7610-11eb-9558-98e10431666e.png)

**Comment by arun11299** [comment]
I am not sure what you mean by it doesn't support C++11. The code does not compile without atleast C++11 standard.

**Comment by vstumpf** [comment]
The README says 
```
Support for C++11 seems trivial based on the changes required. Why not support C+11 then ?

Its 2018 now! If I ever start getting requests to have support for C++11, then I will surely consider it.
```
Is this no longer the case?

**Comment by arun11299** [comment]
Good catch. I really don't remember what is that about :). It certainly uses C++11 features, no doubt. Maybe I meant using C++14/17 std::string_view et al instead of hand rolled one.
